incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -46,7 +46,23 @@ func RunShouldContain(substr string, seconds float64, cmdln ...string) {
 
 func RunShouldContainMultiple(substrs []string, seconds float64, cmdln ...string) {
 	Eventually(func() error {
-		stdout, err := RunCommand(cmdln...)
+		return runShouldContainMultiple(substrs, cmdln...)
+	}, seconds).Should(Succeed())
+}
+
+func RunErrorShouldContain(substr string, seconds float64, cmdln ...string) {
+	RunErrorShouldContainMultiple([]string{substr}, seconds, cmdln...)
+}
+
+func RunErrorShouldContainMultiple(substrs []string, seconds float64, cmdln ...string) {
```

Comment:
This doesn't look right to me? `runShouldContainMultiple` returns an error _both_ if `RunCommand` fails, and _also_ if the text cannot be found. So this function would "pass" if the command _succeeded_ but then _didn't_ find the correct string, which is not what you intend.

Reply:
Thanks for catching it! Fixed the logic